app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue

```diff
@@ -131,7 +131,7 @@ export default {
     },
 
     filterObject(_itemTitle, queryText, item) {
-      return Object.values(item.raw)
+      return Object.values(item)
```

Comment:
Looks like something isn't working here too, but not necessarily with this line. If you create a form that allows drafts, start a submission and save as draft, then view the draft... then add team member, switch to basic bceid and type "DEV_WALTER" nothing happens:
<img width="308" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/c838845b-30fa-4ce4-8e61-badfbf5bd674">
But if you then click the "X" it appears:
<img width="323" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/5b55e965-c665-416e-9944-9b6551d1efe3">

Expected: in dev as soon as you type the last "R" in the BCeID username, it pops up the user info:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/647f0bbe-deef-4828-a792-aa40ceee0dbd)

Reply:
ah yeah, i believe this was intentional as some users using bceid basic were using their personal email addresses. to prevent it from listing all email addresses, we opted to have the user type in the exact username or address.

Reply:
Just to clarify - in the top image I have typed in the entire username but it didn't show the user information. The bottom image is the dev behaviour using the same user.